nuxeo-apb/templates/admin_secret.yml.j2

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
admin_secret/user_secret are backing service credential binding notions. I would not expect to see a template for them in nuxeo-apb.